### PR TITLE
backup packages on S3

### PR DESCRIPTION
**Pull request by piuccio**
once a week

### CODE REVIEW

File: cloudformation/story-packages.json [hunk 1]
```diff
@@ -308,6 +329,186 @@
                 }
             }
         },
+        "DynamoDBDataPipeline": {
+            "Type": "AWS::DataPipeline::Pipeline",
```
Comment (theefer) [inline cloudformation/story-packages.json]: Wow, what's this magic!?

**Comment by theefer** [comment]
@gklopper @tackley @sihil Out of interest, should projects start taking this approach to create their own backups of DynamoDB? Seems scary not to have any backups otherwise (data could be lost due to error, CF mistake, account compromised, even AWS issue).

File: cloudformation/story-packages.json (hunk 1 above)
Comment (piuccio) [inline cloudformation/story-packages.json]: Just translating [this concept](http://docs.amazonaws.cn/en_us/amazondynamodb/latest/developerguide/DynamoDBPipeline.html) in cloudformation.

Comment (theefer) [inline cloudformation/story-packages.json]: Nice!

**Comment by sihil** [comment]
It depends on the value of the data, whether it is a canonical source of truth and how easy it would be to re-create if it was lost. In general I think we should always be backing up data stores containing canonical data. The frequency and retention of the backups comes out of the value of the data. In this case weekly - I'm guessing the assumption is that if a week of data was lost it wouldn't take long to re-create it.

**Comment by theefer** [comment]
Makes sense to me.

cc @kenoir I think we should do the same for our metadata, collections and metadata tables..

**Pull request merged**